integ-test/src/test/java/org/opensearch/sql/legacy/JoinIT.java

```diff
@@ -288,6 +288,7 @@ public void hintMultiSearchCanRunFewTimesNL() throws IOException {
     Assert.assertThat(hits.length(), equalTo(42));
   }
 
+  @Ignore
```

Comment:
Should we put comment to indicate why it is ignored? (or maybe TODO comment?)

Reply:
added comment